aws/resource_aws_wafv2_web_acl.go

```diff
@@ -102,9 +102,9 @@ func resourceAwsWafv2WebACL() *schema.Resource {
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
-									"allow": wafv2EmptySchema(),
-									"block": wafv2EmptySchema(),
-									"count": wafv2EmptySchema(),
+									"allow": wafv2AllowConfigSchema(),
+									"block": wafv2BlockConfigSchema(),
+									"count": wafv2CountConfigSchema(),
```

Comment:
let's also add these newly defined blocks in `wafv2_web_acl.html.markdown`

Reply:
These have been added to the markdown. Let me know if I did it correctly or not :-)